File: Solved/11764.py
'''
*
* Problema UVA 11764 - Jumping Mario
*
'''
import sys

cases = int(sys.stdin.readline())
c = 0

for i in range(cases):
	c += 1
	nWalls = int(sys.stdin.readline())
	array = sys.stdin.readline().strip().split(' ')
	high = 0
	low = 0

	for j in range(len(array)-1):
		if( int(array[j]) < int(array[j+1]) ):
			high += 1
		elif( int(array[j]) > int(array[j+1]) ):
			low += 1

	print("Case {0}: {1} {2}".format(c, high, low))

File: Solved/11764.c
/*
*
* Problema UVA 11764 - Jumping Mario
*
*/

#include <stdio.h>
#include <stdio.h>

int main(){
	int cases, cWalls, walls[50], i, high, low, c = 0;

	scanf("%d", &cases);

	while(cases--){
		scanf("%d", &cWalls);
		high = 0;
		low = 0;
		c++;

		for(i = 0; i < cWalls; i++){
			scanf("%d", &walls[i]);
		}

		for(i = 0; i < cWalls-1; i++){
			if(walls[i] < walls[i+1]){
				high++;
			}else if(walls[i] > walls[i+1]){
				low++;
			}
		}

		printf("Case %d: %d %d\n", c, high, low);
	}


	return 0;
}